CHANGELOG.md

```diff
@@ -10,14 +10,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+[v0.11.0[[v0.11.0_log]
+--------------
+
 ### Added
-- OE SDK Release packages can now be built on non-SGX and non-FLC machines.
+- Open Enclave SDK Release packages can now be built on non-SGX and non-FLC machines.
 - Support for arbitrarily large thread-local data for SGX machines.
+- Experimental support for OpenSSL inside enclaves has been added while building the SDK from source.
+     - Use BUILD_OPENSSL flag while compiling the SDK
+     - [OpenSSLSupport.md](docs/OpenSSLSupport.md) documents supported options and configuration needed to OpenSSL inside an enclave.
 
```

Comment:
Please add these:

- Custom claims buffer serialization/de-serialization helper functions
- `oe_verify_evidence()` and `oe_verify_attestation_certificate_with_evidence()` added to host-verify package
- SGX attestation endorsement claims from `oe_verify_evidence()`
  - `OE_CLAIM_SGX_TCB_INFO`
  - `OE_CLAIM_SGX_TCB_ISSUER_CHAIN`
  - `OE_CLAIM_SGX_PCK_CRL`
  - `OE_CLAIM_SGX_ROOT_CA_CRL`
  - `OE_CLAIM_SGX_CRL_ISSUER_CHAIN`
  - `OE_CLAIM_SGX_QE_ID_INFO`
  - `OE_CLAIM_SGX_QE_ID_ISSUER_CHAIN`

Reply:
I changed the wording a little. Can you please verify that it is accurate?